virttest/env_process.py

```diff
@@ -1030,6 +970,15 @@ def preprocess(test, params, env):
     if params.get("not_preprocess", "no") == "no":
         process(test, params, env, preprocess_image, preprocess_vm)
 
+    # change the kernel params if configured
+    if kernel_extra_params_add or kernel_extra_params_remove:
+        for vm in env.get_all_vms():
+            if vm:
+                if not vm.is_alive():
+                    vm.start()
```

Comment:
I run a case with `kernel_extra_params_add` and `kernel_extra_params_remove` recently for qemu, Found a error: `vm has no attribute start()`. Check the codes, there is no method start() in VM class in qemu_vm.py. So I thought that is not support in qemu test.
Thanks.

Reply:
For qemu tests, by this time in env_process VMs would have created and there is no concept of destroy/start so I will include additional check for libvirt test. It will eventually supported for qemu test, however the reboot after kernel param change is taken care in `update_boot_option()`.

Thanks for reporting it, please help review https://github.com/avocado-framework/avocado-vt/pull/2131